impeller/typographer/typographer_unittests.cc

```diff
@@ -375,6 +375,58 @@ TEST_P(TypographerTest,
   ASSERT_NE(old_packer, new_packer);
 }
 
+TEST(TypographerTest, CanCloneRectanglePackerEmpty) {
+  auto skyline = RectanglePacker::Factory(256, 256);
+
+  EXPECT_EQ(skyline->percentFull(), 0);
```

Comment:
In a later patch, we should consider a refactor to use PascalCase like in the rest of the engine.

Reply:
I just did that here, it was easy enough